### ISSUE TITLE
Social share text is missing from shared links

### ISSUE THREAD

**Issue by thepegisin** [cagov/covid19#375]
Text under share links: When you share the link to covid19.ca.gov (via text message, or tweet, or a million other ways) you used to get a bit of explanatory text under it (something like "we're working hard to respond to keep CA healthy").

First issue: That text was in English, even when you were sharing the link to a translated version of the site.

Second issue: Now that text is gone completely, which fixes the translation issue, but deprives the user of helpful context text. 

**TL;DR: Can we put back the helpful share text and have it be translated to match the language of the site shared?**

**Comment by aaronhans** [comment]
<img width="507" alt="Screen Shot 2020-04-28 at 3 00 11 PM" src="https://user-images.githubusercontent.com/353360/80541966-05c74580-8961-11ea-8895-b35d8696ac48.png">

verified social share description is now present and translated using twitter card validator pictured in screenshot

**Comment by aaronhans** [comment]
code change here https://github.com/cagov/covid19/pull/384

**Comment by thepegisin** [comment]
Thanks, @aaronhans!
